akka-typed-tests/src/test/scala/akka/typed/persistence/scaladsl/PersistentActorCompileOnlyTest.scala

```diff
@@ -401,16 +405,16 @@ object PersistentActorCompileOnlyTest {
     PersistentActor.immutable[Command, Event, State](
       persistenceId = "myPersistenceId",
       initialState = (),
-      actions = Actions { (_, cmd, _) ⇒
+      commandHandler = CommandHandler { (_, cmd, _) ⇒
         cmd match {
           case Enough ⇒
-            Persist(Done)
-              .andThen(
-                SideEffect(_ ⇒ println("yay")),
-                Stop())
+
+            Effect.persist(Done)
+              .andThen((_: State) ⇒ println("yay"))
+              .andThenStop
```

Comment:
Struggled a while with a generic method like before, which is hard because `Event` is covariant in `Effect`, then realized that the only public API currently needed is for `stop` and even if we come up with more useful chainables in the future, it is better to enumerate them with concrete methods like this instead and entirely avoid that type system problem.